Sources/SwifQL/SwifQLSelectBuilder.swift

```diff
@@ -156,12 +177,13 @@ public class SwifQLSelectBuilder {
         if orderBy.count > 0 {
             query = query.orderBy(orderBy)
         }
-        if let offset = offset {
-            query = query.offset(offset)
-        }
         if let limit = limit {
             query = query.limit(limit)
         }
+        if let offset = offset {
+            query = query.offset(offset)
+        }
         return query
     }
+    
```

Comment:
Could you please also remove this redundant line 🙂

Reply:
done